rocks-db/src/lib.rs

```diff
@@ -827,3 +809,14 @@ impl Storage {
         }
     }
 }
+
+#[allow(unused_variables)]
+pub trait ToFlatbuffersConverter<'a> {
+    type Target: 'a;
+    fn convert_to_fb(&self, builder: &mut FlatBufferBuilder<'a>) -> WIPOffset<Self::Target> {
+        todo!()
```

Comment:
just out of curiosity, why do we define a default implementation that'll panic in runtime?

Reply:
Ugly patch to preserve backward compatibility. It will now panic at runtime because atm no structures without flatbuffers call those methods. But yes, it's better to be dropped.